stix_shifter_modules/guardium/stix_translation/json/to_stix_map.json

```diff
@@ -1,130 +1,142 @@
 {
-  "Login Succeeded": {
-    "key": "number_observed",
-    "cybox": false,
-    "transformer": "ToInteger"
-  },
-  "Client IP": [
+  "DBUser": [
     {
-      "key": "ipv4-addr.value",
-      "object": "src_ip"
-    }
-  ],
-  "Server IP": [
+      "key": "user-account.user_id",
+      "object":"account" 
+    },
     {
-      "key": "ipv4-addr.value",
-      "object": "dst_ip"
+      "key": "x-ibm-finding.src_database_user_ref",
+      "object": "finding",
+      "references": "account"
     }
-  ],
-  "DB User Name": [
+   ]
+  ,
+ "db_user": [
     {
-      "key": "user-account.user_id"
+      "key": "user-account.user_id",
+      "object":"account" 
     },
     {
-      "key": "x_guardium.db_user",
-      "cybox": false
+      "key": "x-ibm-finding.src_database_user_ref",
+      "object": "finding",
+      "references": "account"
     }
-  ],
-  "OS User": [
+   ]
+  ,
+ "OSUser": [
     {
-      "key": "user-account.user_id"
+      "key": "user-account.login_name",
+      "object":"account"
     },
     {
-      "key": "x_guardium.os_user",
-      "cybox": false
+      "key": "x-ibm-finding.src_database_user_ref",
+      "object": "account",
+      "references": "user_id"
     }
-  ],
-  "Count of Exceptions": {
-    "key": "number_observed",
-    "cybox": false,
-    "transformer": "ToInteger"
-  },
-  "Audit Process Description": {
-    "key": "x_guardium.audit_process_description",
-    "cybox": false
-  },
-  "Audit Task Description": {
-    "key": "x_guardium.audit_task_description",
-    "cybox": false
-  },
-  "DETAIL": {
-    "key": "x_guardium.run_detail",
+   ],
+  "LogSourceId": {
+    "key": "x__custom_property.log_source_id",
     "cybox": false
   },
-  "Event Type": {
-    "key": "x_guardium.event_type",
+  "Magnitude": {
+    "key": "x__custom_property.magnitude",
```

Comment:
You'll want this as `x_guardium.magnitude` since it is a custom property for Guardium.